cypress/apiRequests/Product.js

```diff
@@ -88,20 +94,27 @@ class Product {
     price = 1,
     costPrice = 1
   ) {
+    const channelListings = channelId
```

Comment:
How about we make a utils function out of this e.g. getValueWithDefault? I see it could look like
```suggestion
    const getValueWithDefault = (condition, value, default = ") => !!condition ? value : default
    const channelListings = getValueWithDefault(channelId, `channelListings:{...`)
```

Guess it could come in pretty handy in tests